# MoveLinear.srv

# Request
float64[] pose1
float64[] pose2
float64 velocity
float64 acceleration
---
# Response
tf2_msgs/TFMessage transform